storage/blockdevice/COMPONENT_QSPIF/source/QSPIFBlockDevice.cpp

```diff
@@ -1099,6 +1102,15 @@ int QSPIFBlockDevice::_handle_vendor_quirks()
             tr_debug("Applying quirks for ISSI");
             _num_status_registers = 1;
             break;
+        case 0x01:
+            if (vendor_device_ids[1] == 0x02 && vendor_device_ids[2] == 0x20) {
```

Comment:
All the other cases emit a trace debug message to say we're applying the "quirk". Should we also add this to our new case?

Reply:
Good idea

Reply:
Done